tests/features/custom_page/collection.custom_page.feature

```diff
@@ -0,0 +1,44 @@
+@api
+Feature: "Add custom page" visibility options.
+  In order to manage custom pages
+  As a collection member
+  I need to be able to add "Custom Page" content through UI.
+  Background:
+    Given users:
+    | name            | mail                  | roles         | pass |
+    | Rufus Drumknott | member@example.com    |               | test |
```

Comment:
Is it normal to have the roles empty? Also if you don't mind it would be nice to not reuse test data from other tests (like the username "Rufus Drumknott"). It's better to test with unique data so we have more chances of catching problems.